packages/compass-global-writes/src/index.ts

```diff
@@ -28,6 +28,6 @@ const CompassGlobalWritesHadronPlugin = registerHadronPlugin(
 export const CompassGlobalWritesPlugin = {
   name: 'GlobalWrites' as const,
   provider: CompassGlobalWritesHadronPlugin,
-  content: GlobalWrites,
-  header: GlobalWritesTabTitle,
+  content: GlobalWrites as React.FunctionComponent,
+  header: GlobalWritesTabTitle as React.FunctionComponent,
```

Comment:
do we need these assertions?

Reply:
hm, good point. so the current setup is such that all the plugins registered with `registerHadronPlugin` get same props defined by `CollectionTabPluginMetadata`. For this plugin, as i am not adding those and when connecting with redux, the [ts check fails with this error](https://spruce.mongodb.com/task/10gen_compass_main_check_check_patch_df47e20728978e1544dccaef9251273df0081782_66f55f9463a74c0007777c7b_24_09_26_13_20_21/logs?execution=0).